### PR TITLE
Update issue manager with "feedback requested" behavior

### PR DESCRIPTION
**Pull request by mfisher87**
We weren't using an "answered" label, so I just replaced it.

Triggers updated per documentation!

### CODE REVIEW

**Comment by mfisher87** [comment]
I specifically want to use this functionality for #356

File: .github/workflows/issue-manager.yml [hunk 1]
```diff
@@ -2,20 +2,28 @@ name: "Issue Manager"
...
+    types:
+      - "labeled"
+  pull_request_target:
+    types:
+      - "labeled"
 
 jobs:
   issue-manager:
-    runs-on: ubuntu-latest
+    runs-on: "ubuntu-latest"
```
Comment (mfisher87) [inline .github/workflows/issue-manager.yml:18]: I feel strongly that quoting YAML strings is important. There's too many weird things that can happen if you don't. Just like bash :) Is there a pre-commit validator that can check for this? :)

Comment (jhkennedy) [reply .github/workflows/issue-manager.yml:18]: GitHub's own documentation doesn't quote simple YAML strings, except for `${{}}` contexts, e.g.,
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on

Other than a few special cases (number-y strings and special characters) I've not had any issues with unquoted strings in YAML. So, I feel it's entirely unnecessary and would prefer to match GitHub's documentation examples as much as possible.

Comment (mfisher87) [reply .github/workflows/issue-manager.yml:18]: I don't think this is something with an objectively correct answer, but I also don't really consider GitHub an authority on YAML quality, so I'm not terribly swayed in my opinion by that :) One of the strange thing YAML does is convert the value `no` (the word "no" or the country code for Norway) to `False`. String rules in YAML are very complicated. I think "explicit is better than implicit", and "simple is better than complex" are my guiding principles in this preference.

If you feel strongly against adding the quotes, I don't think we're going to resolve that. I'm happy to restore this to a mix of quoted and unquoted stings as it was before so we can get this merged.

Comment (mfisher87) [reply .github/workflows/issue-manager.yml:18]: Following up on the earlier thought about validation, there is a yamllint rule that can enforce always-quoted strings (single or double), never enforce quotes, or permit quotes only when needed (for all the special cases like `no`, `yes`, `true`, `false`, `#`, `[]`, `{}`, `\d.*`... probably more I'm not aware of). If/when we have consensus on this issue, I think it'd be great to record that decision as a linting rule.

https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.quoted_strings

Comment (jhkennedy) [reply .github/workflows/issue-manager.yml:18]: I don't think my preference is strong enough to warrant a standoff :rofl: 

I've just rarely hit this issue in practice, and almost always with trailing zeros on number-y things. So I see a molehill here, but if you've climbed a mountain and planted a flag, that's cool with me -- we'll go with that.

I do like the idea of making a decision and letting the linter-bot be the sheriff, though!

**Review by jhkennedy — DISMISSED** [review]

File: .github/workflows/issue-manager.yml [hunk 2]
```diff
@@ -2,20 +2,28 @@ name: "Issue Manager"
...
     steps:
-    - uses: tiangolo/issue-manager@master
+    - uses: "tiangolo/issue-manager@master"
```
Comment (jhkennedy) [inline .github/workflows/issue-manager.yml:20]:
```suggestion
    - uses: "tiangolo/issue-manager@0.4.0"
```
It's probably worth pinning to the latest release and using dependabot to keep the action up to date by adding `github-actions`  to it like:
https://github.com/ASFHyP3/hyp3-sdk/blob/develop/.github/dependabot.yml

File: .github/workflows/issue-manager.yml (hunk 1 above)
Comment (mfisher87) [reply .github/workflows/issue-manager.yml:18]: >I do like the idea of making a decision and letting the linter-bot be the sheriff, though!

Yes! Having software solve these problems is much better than dealing with competing preferences. There are lots of usually-not-a-big-deal things people can have strong opinions about, and for me quoting strings in YAML and Bash is one of them. I've been bitten too many times, and "just quote strings" keeps my cognitive load low :) But I don't want things that help me to cause harm to anyone else, so... yeah, let's make this a discussion about linters/formatters :)

File: .github/workflows/issue-manager.yml (hunk 2 above)
Comment (mfisher87) [reply .github/workflows/issue-manager.yml:20]: Agree, good catch.

**Comment by mfisher87** [comment]
Looks to be working:

```
INFO:root:Not clossing issue: #356 as the delay hasn't been reached: 10 days, 0:00:00
INFO:root:Finished
```

https://github.com/nsidc/earthaccess/actions/runs/6965375261/job/18953956336

**Pull request merged**